### ISSUE TITLE
the real prediction power of models

### ISSUE THREAD

**Issue by shenwanxiang** [PatWalters/solubility#1]
Hi, Walters!
Thanks for this job! I understand the Graph-based methods are usually easy to be over-fitting, So currently all of these methods are not really better than the transitional linear regression models in prediction solubility, right?

**Comment by PatWalters** [comment]
Take a look at the final plot in this blog post
https://practicalcheminformatics.blogspot.com/2018/09/predicting-aqueous-solubility-its.html
Note that the error bars on the correlations for all of the models overlap.   In this particular case, we don't see a significant difference between models.  This actually has nothing to with overfitting.  Even with a neural network, we can employ methods such as dropout which will enable us to avoid overfitting.
